### PR TITLE
PR: Create dependabot.yml file fixes issue #146

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] Create `dependabot.yml` file fixes issue #146

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: mix
```
Comment (SimonLab) [inline .github/dependabot.yml:3]:
```suggestion
- package-ecosystem: npm
```

**Review by SimonLab — COMMENTED** [review]
I think we need to switch the value to npm for package-ecosystem

**Comment by nelsonic** [comment]
@SimonLab good catch. Thanks.

**Review by SimonLab — APPROVED** [review]
:+1:

**Pull request merged**